.github/workflows/ci.yml

```diff
@@ -1,11 +1,7 @@
 name: CI
 
 on:
-  push:
-    branches:
-      - '**'
-  pull_request:
-    branches: [ main ]
+  - push
```

Comment:
Curious as to why you changed this?

Reply:
Because the CI was not triggered in the default setting.